### PR TITLE
Add embed example.

### LINKED ISSUE #739: [HTML Examples] Embedded content ~ embed
Need an HTML example for the `embed` element
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/embed

## Contributing

https://github.com/mdn/interactive-examples/blob/master/CONTRIBUTING-HTML.md

### PR DESCRIPTION
**Pull request by sheeeng**
Fix #739.

### CODE REVIEW

**Comment by sheeeng** [comment]
I did not manage to disable autoplay video inside the `<embed>` element.
Tried `autoplay="false"` and `autostart="false'`, but to no avail.

Any suggestions to disable autoplay of video inside the `<embed>` element?

**Comment by wbamberg** [comment]
@sheeeng , thanks again for your contribution! It looks great.

> Any suggestions to disable autoplay of video inside the <embed> element?

I don't know how to do this either. Maybe @schalkneethling does?

I think it would be better to take a copy of the video under /media/examples, rather than stream it from another site. I think perhaps the one here http://techslides.com/sample-webm-ogg-and-mp4-video-files-for-html5 would be a good choice, for a few reasons:

* it's quite small (the webm is the smallest so perhaps we should choose that)
* it's provided as a sample to use
* it's short and not annoying, so I think it would be acceptable even if we can't prevent autoplay

What do you think?

File: live-examples/html-examples/embedded-content/css/embed.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    font-size: .8rem;
+    position: absolute;
+    bottom: 10px;
+    right: 10px;
+}
+
+.output {
+    background: #eee;
+    position: relative;
+}
```
Comment (wbamberg) [inline live-examples/html-examples/embedded-content/css/embed.css]: Please make sure files end with a new line.

**Comment by sheeeng** [comment]
Pre-commit hooks failed. Any idea?

```
husky > npm run -s precommit (node v8.4.0)

 PASS  __tests__/console-utils.test.js

 RUNS  __tests__/puppeteer-js-editor.test.js

Test Suites: 1 passed, 1 of 2 total
Tests:       12 passed, 12 total
 FAIL  __tests__/puppeteer-js-editor.test.js (5.371s)
  ● JS Editor › renders expected output after clicking Run

    Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.

      at node_modules/jest-jasmine2/build/queue_runner.js:68:21

Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 failed, 12 passed, 13 total
Snapshots:   0 total
Time:        6.14s
Ran all test suites.

husky > pre-commit hook failed (add --no-verify to bypass)
```

**Comment by schalkneethling** [comment]
@sheeeng make sure there is no other instance of the dev server running. Also make sure you have installed all the latest things by running `npm install`

**Comment by sheeeng** [comment]
@schalkneethling, thanks! It works now after `npm install && npm run build && npm start` _and then_ browse `http://localhost:4444/pages/tabbed/embed.html`.

File: live-examples/html-examples/embedded-content/embed.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<embed
+    type="video/webm"
+    src="/media/examples/small.webm"
+    width="320"
+    height="240">
```
[the remark this answers is not in the document]
Comment (sheeeng) [reply live-examples/html-examples/embedded-content/embed.html]:
```diff
--- a/live-examples/html-examples/embedded-content/embed.html
+++ b/live-examples/html-examples/embedded-content/embed.html
@@ -1,6 +1,8 @@
 <embed
     type="video/webm"
     src="/media/examples/small.webm"
+    autoplay="false"
+    autostart="false"
     width="320"
     height="240">
```
Tried both `"false"` and `"0"`, but the video still playing automatically.

[the remark this answers is not in the document]
Comment (wbamberg) [reply live-examples/html-examples/embedded-content/embed.html]: Yes, I found that post when reviewing this PR, but none of those worked for me either. I think given a short, non-annoying video (which this is) then autoplay is acceptable. But I'd like to get @schalkneethling opinion too, before merging this.

[the remark this answers is not in the document]
Comment (sheeeng) [reply live-examples/html-examples/embedded-content/embed.html]: I have uploaded another video example with WebM and MP4 format at #885. Perhaps, we can decide which video example to use in both places?

**Comment by sheeeng** [comment]
Use audioless video for the example.
Folks can remove the trailing `_audioless` on the filename to try the video with audio.
Is this approach better?

**Pull request merged**